idea_town/frontend/static-src/app/views/details-view.js

```diff
@@ -0,0 +1,83 @@
+import Model from 'ampersand-model';
+import Collection from 'ampersand-collection';
+import CollectionRenderer from 'ampersand-collection-view';
+
+import BaseView from './base-view';
+import template from '../templates/details-view';
+import ContributorView from './contributor-view';
+
+const Contributors = Collection.extend({
+  model: Model.extend({
+    extraProperties: 'allow'
+  })
+});
+
+export default BaseView.extend({
+  _template: template,
+
+  bindings: {
+    'model.modified_date': {
+      type: function(el, value) {
```

Comment:
Actually, you could use an arrow function here, which I think skips the function naming issue too.